src/main/java/org/entur/gbfs/loader/v2/GbfsV2Subscription.java

```diff
@@ -108,6 +125,10 @@ public void update() {
       );
       consumer.accept(delivery);
     }
+
```

Comment:
Put this afterUpdate in a finally block for a try block around the update call (same for v3)